src/Illuminate/Contracts/Auth/Guard.php

```diff
@@ -21,7 +21,7 @@ public function guest();
     /**
      * Get the currently authenticated user.
      *
-     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     * @return \Illuminate\Contracts\Auth\Authenticatable|\App\User|null
```

Comment:
As I said, this change won't make any difference if the User model is somewhere else. IDE will not find the class and provide completion from only the Authenticatable contract.

It has no side effects as far as I tested(have you encountered any?) and it makes life a little bit easier for those who stick to the defaults.